### ISSUE TITLE
Context 'TSInterfaceDeclaration' not working with 'export' keyword

### ISSUE THREAD

**Issue by medeman** [gajus/eslint-plugin-jsdoc#385]
I configured the `jsdoc/require-jsdoc` rule as follows:

```json
{
  ...
  "rules": {
    "jsdoc/require-jsdoc": ["warn", {
      "require": {
        "ArrowFunctionExpression": true,
        "ClassDeclaration": true,
        "ClassExpression": true,
        "FunctionDeclaration": true,
        "FunctionExpression": true,
        "MethodDefinition": true
      },
      "contexts": [
        "ClassProperty",
        "TSInterfaceDeclaration",
        "TSPropertySignature"
      ]
    }]
  }
  ...
}
```

To my understanding, the `TSInterfaceDeclaration` context should require that interfaces have a JSDoc comment. This works just fine when used on an interface normally, i.e.:

```typescript
/**
 * This example interface is great!
 */
interface Example {
  /**
   * My super test string!
   */
  test: string
}
```

However, when prepending the `export` keyword, the JSDoc comment is not detected and an error message is shown:

```typescript
/**
 * This example interface is great!
 */
export interface Example {
  /**
   * My super test string!
   */
  test: string
}
```

This example results in `eslint: jsdoc/require-jsdoc - Missing JSDoc comment.` for the interface declaration.

Am I doing something wrong or is this a bug/oversight? It works fine with `export class` for example, just not with `export interface`.
